### PR TITLE
NetStandard Tests Before Deprecation

### PR DESCRIPTION
**Pull request by MiYanni**
## Description

'Before Deprecation' set of Az tests from this issue: #6931

Requires this PR before Travis will pass: https://github.com/Azure/azure-powershell/pull/7495

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by MiYanni** [comment]
@azuresdkci Retest this please

File: src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/IntegrationAccountGeneratedIcnTests.cs [hunk 1]
```diff
@@ -37,7 +37,7 @@ public IntegrationAccountGeneratedIcnTests(Xunit.Abstractions.ITestOutputHelper
         /// </summary>
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void TestGetIntegrationAccountGeneratedIcnWithoutAgreementType()
+        public void TestGetIAGeneratedIcnWithoutAgreementType()
```
Comment (markcowl) [inline src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/IntegrationAccountGeneratedIcnTests.cs]: Do we need to change the recording json file path?

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/PolicyTests.cs [hunk 2]
```diff
@@ -31,7 +31,11 @@ public PolicyTests(Xunit.Abstractions.ITestOutputHelper output)
             XunitTracingInterceptor.AddToContext(_logger);
         }
 
+#if NETSTANDARD
+        [Fact(Skip = "Backup tests require multiple versions of the same SDKs, not supported in NetStandard")]
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/PolicyTests.cs]: Let's file an issue for this - the solution is to update the tests (better) or to  compile the generated code for one of the SDKs into the solution

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs [hunk 3]
```diff
@@ -37,7 +37,11 @@ public void TestRecoveryServicesVaultCRUD()
             TestController.NewInstance.RunPsTest(_logger, "Test-RecoveryServicesVaultCRUD");
         }
 
+#if NETSTANDARD
+        [Fact(Skip = "Different parameter set used for NetStandard. Cannot process command because of one or more missing mandatory parameters: Certificate.")]
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs]: Let's file an issue for them to fix this.

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/PolicyTests.cs (hunk 2 above)
Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/PolicyTests.cs]: Created: https://github.com/Azure/azure-powershell/issues/7533

File: src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/IntegrationAccountGeneratedIcnTests.cs (hunk 1 above)
Comment (MiYanni) [reply src/ResourceManager/LogicApp/Commands.LogicApp.Test/ScenarioTests/IntegrationAccountGeneratedIcnTests.cs]: For net452, yes. That was changed here: https://github.com/Azure/azure-powershell/pull/7497/files#diff-2ba361f18e7b81bc1187080711233190R186

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs (hunk 3 above)
Comment (MiYanni) [reply src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs]: Created: https://github.com/Azure/azure-powershell/issues/7534

**Pull request merged**